src/components/modal/modal.js

```diff
@@ -600,11 +600,26 @@ export const BModal = /*#__PURE__*/ Vue.extend({
     onClose() {
       this.hide('headerclose')
     },
-    onEsc(evt) {
+    onKeydown(evt) {
       // If ESC pressed, hide modal
       if (evt.keyCode === KeyCodes.ESC && this.isVisible && !this.noCloseOnEsc) {
         this.hide('esc')
       }
+
+      // if BACKSPACE is pressed and we are not inside a textarea or input
+      // prevent default behavior of navigating to the previous page and hide the modal
+      if (
+        evt.keyCode === KeyCodes.BACKSPACE &&
+        (evt.target || evt.srcElement).tagName !== 'TEXTAREA' &&
+        (evt.target || evt.srcElement).tagName !== 'INPUT' &&
+        this.isVisible
```

Comment:
User could also have a content editable area (non form-input, such as a WYSIWYG editor) which would close the modal if the user is backspacing over a character in the content editable div (or other element).

Reply:
Good catch, thank you!